snapcraft.yaml

```diff
@@ -353,6 +353,10 @@ parts:
     build-packages:
       - libidn2-0-dev
       - libunistring-dev
+    override-pull: |
+      set -eux
+      craftctl default
+      sed -i 's#!/usr/bin/env python#!/usr/bin/env python3#g' src/psl-make-dafsa
```

Comment:
I'd be more confident in using patches here, as those are more targetting files.

So just include a patch file and use `patch -p1 < $file` to apply it.

Reply:
Ok.

Reply:
Mmm... how can I apply a patch that is outside the LXC container, in the snapcraft.yaml folder? In the documentation it says to use $SNAPCRAFT_STAGE, but that variable isn't set. Also, CRAFT_STAGE is set, but obviously points to /root/stage, inside the LXC container...

Reply:
I think for this type of change a sed is fine, we do this in many cases within our desktop snaps.